src/layers/HandicapLayer/HandicapLayer.js

```diff
@@ -14,43 +14,75 @@ import LayerHelper from '../layerHelper';
  * @inheritdoc
  */
 class HandicapLayer extends VectorLayer {
-  static getIconStyle(geometry, isHighlighted = false) {
+  static getIconStyle(geometry, isHighlighted = false, barrierfree) {
+    if (typeof barrierfree === 'undefined') {
+      return [
```

Comment:
What about a style cache?

Reply:
style cache added

Reply:
Not sure if this has an effect since the geometry always changes